File: src/me/tcklpl/naturaldisaster/util/PlayerUtils.java
package me.tcklpl.naturaldisaster.util;

import org.bukkit.entity.Player;
import org.bukkit.potion.PotionEffect;

public class PlayerUtils {

    public static void healPlayer(Player p) {
        p.setHealth(20);
        p.setFoodLevel(20);
        p.setFireTicks(0);
        p.setFallDistance(0);
        for (PotionEffect pe : p.getActivePotionEffects())
            p.removePotionEffect(pe.getType());
    }
}

File: src/main/java/me/tcklpl/naturaldisaster/util/PlayerUtils.kt
package me.tcklpl.naturaldisaster.util

import org.bukkit.entity.Player

object PlayerUtils {

    fun healPlayer(p: Player) {
        p.health = 20.0
        p.foodLevel = 20
        p.fireTicks = 0
        p.fallDistance = 0f
        for (pe in p.activePotionEffects) p.removePotionEffect(pe.type)
    }
}
